### PR TITLE
Submitting lesson02 (complete) and lesson03 (first two assignments) for John Rogers

### PR DESCRIPTION
**Pull request by johnroge**
Last attempt at a pull request broke something/ was out of sync; followed github directions for syncing a fork and attempting to re-submit work for lesson02.

### CODE REVIEW

**Comment by natasha-aleksandrova** [comment]
@johnroge looks better, make sure you sync your fork each time (it was mentioned in the original setup instructions)

**Comment by natasha-aleksandrova** [comment]
@charlesrobison original submission: https://github.com/UWPCE-PythonCert-ClassRepos/Self_Paced-Online/pull/1217

**Comment by johnroge** [comment]
Attempting another pull request for the first assignment in Lesson03; do I need to wait for this request to get merged and submit another, or will it get pulled in by this request? Or is this current pull request waiting on something from my end to be merged/ completed?

**Comment by natasha-aleksandrova** [comment]
@johnroge it's waiting to get reviewed; all of your commits will end up under this one until this is merged in.

**Comment by johnroge** [comment]
Thank you, I think I understand now. Please give me a heads up if I'm doing something to make it worse or throw it off again.

File: students/john_rogers/lesson03/list_lab.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    print('Series 1 list with items spelled backwards:')
+    print(fruit_copy)
+
+    original.pop()
+    print('Original list with last item removed: ')
+    print(original)
+
+
+if __name__ == '__main__':
+    main()
```
Comment (charlesrobison) [inline students/john_rogers/lesson03/list_lab.py]: Looks great!

**Pull request merged**